pkg/process/checks/enabled_checks_windows_test.go

```diff
@@ -0,0 +1,74 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build windows
+
+package checks
+
+import (
+	"testing"
+
+	"github.com/DataDog/datadog-agent/pkg/config"
+	configmock "github.com/DataDog/datadog-agent/pkg/config/mock"
+	"github.com/DataDog/datadog-agent/pkg/util/flavor"
+)
+
+func TestProcessCheckWindows(t *testing.T) {
+	deps := createDeps(t)
+	originalFlavor := flavor.GetFlavor()
+	defer flavor.SetFlavor(originalFlavor)
+
+	// Make sure process checks run on the process agent even
+	// when run in core agent mode is enabled
+	t.Run("run in core agent", func(t *testing.T) {
```

Comment:
Minor: this name is a bit misleading since it does not run in the core agent, maybe rename to something like "run in core agent ignored"?

Reply:
Done